core/src/helpers.rs

```diff
@@ -324,6 +331,17 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_bytes_to_felt_vec_and_vec_felt_to_bytes() {
+        let input = Bytes::from(vec![
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 1,
+        ]);
+        let expected_output = vec![FieldElement::from(1_u64)];
```

Comment:
here we'd want to do Bytes(0,0,0,0,0,...,1) -> vec![0,0,0,0...,1] -> vec![felt!(0), felt!(0), ..., felt!(1)]